x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/analysis/AnalysisUtils.java

```diff
@@ -76,10 +76,8 @@ private static Attribute handleSpecialFields(UnresolvedAttribute u, Attribute na
         if (named instanceof FieldAttribute fa) {
 
             // incompatible mappings
-            if (fa.field() instanceof InvalidMappedField) {
-                named = u.withUnresolvedMessage(
-                    "Cannot use field [" + fa.name() + "] due to ambiguities being " + ((InvalidMappedField) fa.field()).errorMessage()
-                );
+            if (fa.field()instanceof InvalidMappedField field) {
```

Comment:
Missing an extra space here
```suggestion
            if (fa.field() instanceof InvalidMappedField field) {
```

Reply:
@dakrone thank you for reviewing and catching the typos. It's a known issue in Spotless and, for the moment, the options are:
1. Accept it
2. Don't use that syntax
3. Use `// tag:noformat` and `// end::noformat`

In this case I chose to leave it as is and, when the bug is fixed, hopefully `spotlessJavaCheck` will catch all these typos and fix them in bulk.